src/middle/ssa/ssastorage.rs

```diff
@@ -356,138 +398,141 @@ impl SSAStorage {
         adjacent.sort_by(|a, b| a.0.cmp(&b.0));
         adjacent.iter().map(|x| x.1).collect::<Vec<_>>()
     }
+}
 
-    pub fn is_block(&self, node: NodeIndex) -> bool {
+/// ////////////////////////////////////////////////////////////////////////////
+/// / Implementation of CFG for SSAStorage.
+/// ////////////////////////////////////////////////////////////////////////////
+
+impl CFG for SSAStorage {
+	type ActionRef = <SSAStorage as Graph>::GraphNodeRef;
+	type CFEdgeRef = <SSAStorage as Graph>::GraphEdgeRef;
```

Comment:
Just a small nitpick, ensure that the indentation is consistent (and 4 spaces).There is a tab here.

Reply:
Hah, it will be a good chance to re-indent the whole codebase, for replacing tab by 4 spaces